### PR TITLE
Move to CNP fees and payments (non prod only)

### PR DESCRIPTION
**Pull request by timja-kainos**
Somehow their certs expired and no one renewed them, migrating to cnp early to resolve

### CODE REVIEW

File: infrastructure/prod.tfvars [hunk 1]
```diff
@@ -7,4 +7,3 @@ ga_tracking_id = "UA-97111056-2"
 external_host_name = "www.moneyclaims.service.gov.uk"
 feature_return_error_to_user = "false"
 capacity = "2"
-
```
Comment (geek-soft) [inline infrastructure/prod.tfvars]: Can we use CNP fees API in prod as well?

**Comment by pragnesh-patel** [comment]
Approved @timja-kainos but some checks were not successful

File: infrastructure/prod.tfvars (hunk 1 above)
Comment (jaleen) [reply infrastructure/prod.tfvars]: yup, it's over https using certs which have CA errors at the moment which should be acceptable

**Comment by timja-kainos** [comment]
@pragnesh-patel that's a unit test failure being worked on by ellie, its related to dates and it isn't working in july for some reason

**Pull request merged**